modular_skyrat/modules/extra_tools/code/datums/components/crafting/recipes/recipes_misc.dm

```diff
@@ -0,0 +1,62 @@
+/datum/crafting_recipe/makeshift/crowbar
+	name = "Makeshift Crowbar"
+	tool_paths = list(/obj/item/hammer/makeshift)
+	result = /obj/item/crowbar/makeshift
+	reqs = list(/obj/item/stack/sheet/iron = 4,
+				/obj/item/stack/sheet/cloth = 1,
+				/obj/item/stack/cable_coil = 1)
+	time = 120
+	category = CAT_MISC
+
+/datum/crafting_recipe/makeshift/hammer
+	name = "Makeshift Hammer"
+	result = /obj/item/hammer/makeshift
+	reqs = list(/obj/item/stack/cable_coil = 1,
+				/obj/item/stack/rods = 2,
+				/obj/item/kitchen/rollingpin = 2)
+	time = 80
+	category = CAT_MISC
```

Comment:
What even is the purpose of a hammer...?

Reply:
To be used for the other recipes.